client-app/src/desktop/tabs/other/OtherTab.js

```diff
@@ -27,8 +28,9 @@ export const otherTab = hoistCmp.factory(
                 {id: 'clock', content: clockPanel},
                 {id: 'customPackage', content: customPackagePanel},
                 {id: 'dateFormats', content: dateFormatsPanel},
-                {id: 'jsx', title: 'Factories vs. JSX', content: jsxPanel},
                 {id: 'errorMessage', title: 'ErrorMessage', content: errorMessagePanel},
+                {id: 'exceptionHandler', title: 'ExceptionHandler', content: exceptionHandlerPanel},
```

Comment:
Maybe call this just "Exception Handling".  Class name itself not really important for applications.